src/Runner.jl

```diff
@@ -917,6 +917,7 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
             csl_paths(host_platform),
             # Add our target/host-specific library directories for compiler support libraries
             target_lib_dir(host_platform),
+            "$host_libdir",
```

Comment:
I'd just add a comment
```suggestion
            # Libdir of the host platform, to run programs in `HostBuildDependency`
            "$(host_libdir)",
            # Add our target/host-specific library directories for compiler support libraries
            target_lib_dir(host_platform),
```